fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -1225,10 +1224,11 @@ private boolean supportPartialPartitionQueryRewriteForExternalTable(Table table)
      *                    the update check.
      * @return          : Collect all need refreshed partitions of materialized view.
      */
-    public Set<String> getPartitionNamesToRefreshForMv(boolean isQueryRewrite) {
+    public boolean getPartitionNamesToRefreshForMv(Set<String> toRefreshPartitions,
```

Comment:
I found the function `getPartitionNamesToRefreshForMv `  is used in many positions. Changing it will cause a lot of conflicts(in the old version).

Can I fix this in the late refactor pr?